framework/packages/abstract-adapter/src/features.rs

```diff
@@ -32,11 +30,6 @@ impl<Error: ContractError, CustomInitMsg, CustomExecMsg, CustomQueryMsg, SudoMsg
     }
 }
 
-impl<Error: ContractError, CustomInitMsg, CustomExecMsg, CustomQueryMsg, SudoMsg> AccountExecutor
```

Comment:
Why was this removed?

Reply:
https://github.com/AbstractSDK/abstract/pull/444#discussion_r1756393019 Check this discussion